src/components/loginform/modules/Container.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from "react-redux";
+import LoginForm from "../LoginForm";
+import { loginRequest } from "../../../actions/authActions";
+
+const mapStateToProps = (state) => ({
+  auth: state.auth,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  loginRequest: (email, password) => dispatch(loginRequest(email, password)),
```

Comment:
Do this in one object. Because in this case if you have 30 actions you will need to pass 30 parameters in component props. But if you pass as object you will pass only 1. and then you can use actions.loginRequest()